client/gutenberg/extensions/prev-next/index.js

```diff
@@ -0,0 +1,75 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import wp from 'wp';
+const { __ } = wp.i18n;
+const { TextControl } = wp.components;
+const { registerBlockType } = wp.blocks;
+
+require( './style.scss' );
```

Comment:
Nit: I'd expect this to be below the last import, but above any module variables - what do you think?

Reply:
🦐

(that means "I tried to add the shrug emoji but then hit enter too quickly assuming another wouldn't appear in place of shrug even though a shrimp came in first")